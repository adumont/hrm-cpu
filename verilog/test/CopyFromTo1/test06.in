54 04
34 00
12 2f
56 61
42 29
54 d8
50 f3
46 d0
36 59
31 b3
53 bd
40 e5
24 7a
10 f4
23 ca
33 22
23 c9
08 b9
06 50
25 ac
38 ea
27 75
42 da
28 8d
13 ed
34 04
58 37
16 3b
36 eb
27 ea
21 2c
50 18
44 0a
22 a5
51 a1
20 50
24 cd
53 97
44 60
09 be
