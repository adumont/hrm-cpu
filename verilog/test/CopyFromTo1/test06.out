04
00
2f
61
29
d8
f3
d0
59
b3
bd
e5
7a
f4
ca
22
c9
b9
50
ac
ea
75
da
8d
ed
04
37
3b
eb
ea
2c
18
0a
a5
a1
50
cd
97
60
be
